apps/gitness/package.json

```diff
@@ -17,8 +17,8 @@
     "test:changed": "vitest --run --changed=origin/main",
     "extract": "i18next --config './i18n.config.ts'",
     "typecheck": "tsc -b",
-    "build:webpack": "rm -rf dist && webpack --config webpack.config.cjs",
-    "start:webpack": "webpack serve --config webpack.config.cjs"
+    "build:webpack": "rm -rf dist && webpack --config webpack.config.cjs --mode=production",
```

Comment:
_Technically_ you don't need this since `production` is the default if nothing is specified, but good to be explicit.